.werft/eks-installer-tests.yaml

```diff
@@ -89,6 +89,8 @@ pod:
       value: "/mnt/secrets/sh-playground-sa-perm/sh-sa.json"
     - name: TF_VAR_dns_sa_creds
       value: "/mnt/secrets/sh-playground-dns-perm/sh-dns-sa.json"
+    - name: TF_VAR_sa_creds
```

Comment:
This is an unrelated change required to re-use the same environment while running tests! Context: https://gitpod.slack.com/archives/C01KLC56NP7/p1664268857397169?thread_ts=1664261197.651119&cid=C01KLC56NP7